packages/swapper/src/swappers/RelaySwapper/utils/relayTokenToAssetId.ts

```diff
@@ -0,0 +1,90 @@
+import type { AssetId, ChainReference } from '@shapeshiftoss/caip'
+import {
+  ASSET_NAMESPACE,
+  ASSET_REFERENCE,
+  CHAIN_NAMESPACE,
+  CHAIN_REFERENCE,
+  toAssetId,
+  toChainId,
+} from '@shapeshiftoss/caip'
+
+import { DEFAULT_RELAY_EVM_TOKEN_ADDRESS } from '../constant'
+import type { RelayToken } from './types'
+
+export const relayTokenToAssetId = (relayToken: RelayToken): AssetId => {
+  const chainReference = relayToken.chainId.toString() as ChainReference
+  const chainId = toChainId({
+    chainNamespace: CHAIN_NAMESPACE.Evm,
+    chainReference,
+  })
+
+  // @TODO: Handle the same for Solana and BTC
+  const isDefaultAddress = relayToken.address === DEFAULT_RELAY_EVM_TOKEN_ADDRESS
+
+  const { assetReference, assetNamespace } = (() => {
+    // @TODO: Handle the same for Solana and BTC
+    if (!isDefaultAddress)
+      return {
+        assetReference: relayToken.address,
+        assetNamespace:
+          chainReference === CHAIN_REFERENCE.BnbSmartChainMainnet
+            ? ASSET_NAMESPACE.bep20
+            : ASSET_NAMESPACE.erc20,
```

Comment:
Unrelated to your PR but always forgetting this and having to monkey-see-monkey-do to get it right is a PITA, summoning @kaladinlight re: do we still need bep20 to be an explicit asset namespace or time for a cleanup?

Reply:
I'll transfer in private but closing this one so we can go forward and merge